test/test_alchemy_language_v1.py

```diff
@@ -10,7 +10,7 @@ def test_api_key(self):
         default_url = 'https://gateway-a.watsonplatform.net/calls'
         inited = watson_developer_cloud.AlchemyLanguageV1(url=default_url, api_key='boguskey',
                                                           x_watson_learning_opt_out=True)
-        assert inited.x_watson_learning_opt_out
+        #assert inited.x_watson_learning_opt_out
```

Comment:
Can delete this line, since it's no longer needed.

Reply:
Done.